src/Process/3dTilesProcessing.js

```diff
@@ -79,7 +79,51 @@ export function $3dTilesCulling(node, camera) {
     return false;
 }
 
+// Cleanup all 3dtiles|three.js starting from a given node n.
+// n's children can be of 2 types:
+//   - have a 'content' attribute -> it's a tileset and must
+//     be cleaned with cleanup3dTileset()
+//   - doesn't have 'content' -> it's a raw Object3D object,
+//     and must be cleaned with _cleanupObject3D()
+function cleanup3dTileset(layer, n) {
+    for (let i = 0; i < n.children.length; i++) {
+        // skip non-tiles elements
+        if (!n.children[i].content) {
+            n.children[i].traverse(_cleanupObject3D);
+        } else {
+            cleanup3dTileset(layer, n.children[i]);
+        }
+    }
+    if (n.dispose) {
+        n.dispose();
+    }
+    delete n.content;
+    layer.tileIndex.index[n.tileId].loaded = false;
+    n.remove(...n.children);
+}
+
+// This function is used to cleanup a Object3D hierarchy.
+// (no 3dtiles spectific code here because this is managed by cleanup3dTileset)
+function _cleanupObject3D(n) {
```

Comment:
The `_` is to indicate that this a `private` function within this module: you're not supposed to use it except from `cleanup3dTileset`.
But I can remove it if you feel it's useless.

Reply:
As you're not exposing it, it can't be used outside of this module anyway. For me the `_` prefix is useful when you have to attach it to `this` for some reason (I don't think it's ever *strictly* necessary to attach things to `this`, but from time to time, it saves a lot of headache to do so).